### DISCUSSION TITLE
Spectating a player who plays a converted beatmap shows the standard beatmap instead of the converted beatmap

### DISCUSSION THREAD

**Discussion by mouzedrift** [ppy/osu#15127] [Q&A] [answered]
i randomly noticed this while spectating in a lobby and this seems to break taiko, mania and ctb converts.

https://user-images.githubusercontent.com/43358824/137577944-f267c8f9-abb0-4805-82b2-6f4624e0b2cb.mp4

**Comment by bdach** [comment, marked as answer]
#15129
